### PR TITLE
{bio}[GCC/6.4.0] HTSlib v1.8

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by boegel — DISMISSED** [review]

**Comment by migueldiascosta** [comment]
@JackPerdue, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

File: easybuild/easyconfigs/h/HTSlib/HTSlib-1.8-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+version = '1.8'
+
+homepage = "http://www.htslib.org/"
+
+description = """
+ A C library for reading/writing high-throughput sequencing data.
+ This package includes the utilities bgzip and tabix
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/h/HTSlib/HTSlib-1.8-GCCcore-6.4.0.eb]: Moved to GCC level

File: easybuild/easyconfigs/c/cURL/cURL-7.59.0-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
+ libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
+ form based upload, proxies, cookies, user+password authentication (Basic,
+ Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
+ and more.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://curl.haxx.se/download/']
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/c/cURL/cURL-7.59.0-GCCcore-6.4.0.eb]: Done

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/a1e42132e2a377b178a4d4d42f4ba807 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/5ab1803dc39ed5ed1a857f1c2a5e1c26 for a full test report.

**Review by boegel — DISMISSED** [review]
lgtm, minimal diff with `HTSlib-1.8-foss-2018a.eb` and `HTSlib-1.8-intel-2018a.eb`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2603.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/c9be59ecf9c2a5c34034ed71da30fe20 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**